Realm/RLMObjectStore.mm

```diff
@@ -73,7 +74,7 @@ static inline void RLMVerifyRealmRead(__unsafe_unretained RLMRealm *const realm)
     }
 }
 
-static inline void RLMVerifyInWriteTransaction(__unsafe_unretained RLMRealm *const realm) {
+inline void RLMVerifyInWriteTransaction(__unsafe_unretained RLMRealm *const realm) {
```

Comment:
Non-static inline in an implementation file doesn't make sense. If this is intended to be called from other files it shouldn't be inline.